language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
+    ```
+    UPDATE `consumer` SET `PASSWORD` = MD5(`PASSWORD`);
+    ```
+
+- Database table names must be plural. 
+    E.g.: 
+    `users`, `subscriptions`
+    
+    for Linking tables :
+    `users_subscriptions`
+
+- Column names in lowercase and must be singular. Avoid using MySQL reserved words such as order, status, etc.
+    E.g.: `username`, `is_active`, `created_by`, `created_at`
+    
+    - Foreign keys must be prefix with parent table.
+    E.g.:
+    `country_id`
+    
+- Table and column names must be enclosed with ` (backtick operator)
+    E.g.:
+    ```
```

Comment:
You can add `sql` after the backticks to [use syntax highlighting](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting):

```sql
SELECT `id`, `name`, `name_sc` FROM `area` WHERE `country_id` = 1 AND `region_id` = 5;
```
as opposed to:
```
SELECT `id`, `name`, `name_sc` FROM `area` WHERE `country_id` = 1 AND `region_id` = 5;
```